pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -834,6 +835,27 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 		}
 	}
 
+	// Validate CPU Feature Policies
+	if spec.Domain.CPU != nil && spec.Domain.CPU.Features != nil {
+		isValidePolicy := func(policy string) bool {
```

Comment:
isValidePolicy -> isValidPolicy?

Reply:
you cant add '?' here, this is bad syntax.

Reply:
i meant isValidPolicy without question mark :)

Reply:
ah , i see what you mean :)